src/components/AddPeopleAndRoles/OrgPerson.vue

```diff
@@ -62,37 +63,46 @@
                     :rules="orgNameRules"
                   />
                 </div>
+
+                <!-- Roles -->
                 <label class="sub-header">Roles</label>
                 <v-row>
                   <v-col cols="4" v-if="isPerson">
-                    <div :class="{'highlightedRole': isRoleLocked(Roles.COMPLETING_PARTY)}">
+                    <div :class="{ 'highlightedRole': isRoleLocked(Roles.COMPLETING_PARTY) }">
                       <v-checkbox
-                        v-model="isCompletingParty"
-                        id='cp-checkbox'
-                        label="Completing Party"
+                        id="cp-checkbox"
+                        v-model="selectedRoles"
+                        :value="Roles.COMPLETING_PARTY"
+                        :label="Roles.COMPLETING_PARTY"
                         :disabled="isRoleLocked(Roles.COMPLETING_PARTY)"
+                        :rules="roleRules"
                         @change="assignCompletingPartyRole()"
                       />
                     </div>
                   </v-col>
                   <v-col cols="4">
-                    <div :class="{ 'highlightedRole':
-                      isRoleLocked(Roles.INCORPORATOR) ||
-                      orgPerson.officer.partyType === IncorporatorTypes.CORPORATION }"
+                    <div :class="{ 'highlightedRole': isRoleLocked(Roles.INCORPORATOR) || isOrg }"
                     >
                       <v-checkbox
-                        v-model="isIncorporator"
+                        v-model="selectedRoles"
+                        :value="Roles.INCORPORATOR"
                         :label="incorporatorLabel"
-                        :disabled="isRoleLocked(Roles.INCORPORATOR) ||
-                        orgPerson.officer.partyType === IncorporatorTypes.CORPORATION"
+                        :disabled="isRoleLocked(Roles.INCORPORATOR) || isOrg"
```

Comment:
"isOrg" already did this check (and used null chaining for safety), so this cleans up the code.